Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/LATISS-Combined-Calibrations-Procedure/latiss-combined-calibrations-procedure.rst

```diff
@@ -193,12 +229,20 @@ An example set of configuration parameters is as follows:
     do_defects: True
     do_ptc: True
 
-Notes
-^^^^^
+Notes for the previous example
```

Comment:
This is just for the previous example? If so, just leave it in the previous section, adding another sort of adds confusion.